test/cases_matching_callbacks/1.callbacks.c

```diff
@@ -0,0 +1,39 @@
+#include <assert.h>
+#include <metababel/metababel.h>
+#include <stdbool.h>
+#include <stdio.h>
+
+struct data_s {
+  uint64_t mcb_count;
+  uint64_t ecb_count;
+};
+
+typedef struct data_s data_t;
+
+void btx_initialize_usr_data(void *btx_handle, void **usr_data) {
+  *usr_data = calloc(1, sizeof(data_t));
+}
+
+void btx_finalize_usr_data(void *btx_handle, void *usr_data) {
+  data_t *data = (data_t *)usr_data;
+  assert(data->ecb_count == 4);
+  assert(data->mcb_count == 4);
```

Comment:
Will be better if we don't have the same count.

Reply:
Really good call !!